Lib/test/test_mimetypes.py

```diff
@@ -145,11 +145,6 @@ def test_guess_all_types(self):
         self.assertNotIn('.no-such-ext', all)
 
     def test_encoding(self):
-        getpreferredencoding = locale.getpreferredencoding
-        self.addCleanup(setattr, locale, 'getpreferredencoding',
-                                 getpreferredencoding)
-        locale.getpreferredencoding = lambda: 'ascii'
-
```

Comment:
This hack doesn't work for most cases.

Reply:
Please remove ``import locale`` at the top.

This code is correct. I don't think that the locale encoding is still used: MimeType.read() calls ``open(filename, encoding="utf-8")`` since Python 3.3: commit 82ac9bcdb3844369fd9274b559fab073a0142b1e.